app/src/main/java/org/p2p/wallet/jupiter/ui/main/widget/SwapWidget.kt

```diff
@@ -124,7 +151,7 @@ class SwapWidget @JvmOverloads constructor(
             newAmountRaw != oldAMountRaw ||
             newTextColorRes != oldTextColorRes
         ) {
-            editTextAmount.bindOrGone(newAmount)
+            editTextAmount.bindOrGone(newAmount, force = true)
```

Comment:
I don't know why it is working in swaps without forced model check, but not here